app/services/food_service.rb

```diff
@@ -0,0 +1,23 @@
+class FoodService
+  def results(q)
+    response = conn.get("/fdc/v1/foods/search?") do |faraday|
+      faraday.params['query'] = q
+    end
+    parsed = JSON.parse(response.body, symbolize_names: true)[:foods]
+
+  end
+
+  def total_results(q)
+    response = conn.get("/fdc/v1/foods/search?") do |faraday|
+      faraday.params['query'] = q
+    end
+    JSON.parse(response.body, symbolize_names: true)[:totalHits]
```

Comment:
If the search term is the same, which I assume that it is based on the spec, then we actually don't need to hit the API twice with this method and the. We should have one service method that returns the raw JSON to the API. This way, we only ever make one request. Then, our facade can dig into both `:foods` and `:total_hits`, and we could return that to the controller as a 2D array. Something like:
`[30,000, [FoodObject1, FoodObject2,...]]` 
Then, we can deconstruct this array with one line  of syntactic sugar in our controller:
`@total_hits, @foods = FoodFacade.results`
The entire benefit being that we only make one API call, and this speeds up our application.